rust/src/lib.rs

```diff
@@ -1606,64 +1607,21 @@ pub unsafe extern "C" fn zcashlc_seed_fingerprint(
     unwrap_exc_or(res, false)
 }
 
-/// Returns the most recent block height to which it is possible to reset the state
-/// of the data database.
+/// Rewinds the data database to at most the given height.
 ///
-/// # Safety
+/// If the requested height is greater than or equal to the height of the last scanned block, this
+ * function sets the `safe_rewind_ret` output parameter to `-1` and does nothing else.
```

Comment:
```suggestion
/// function sets the `safe_rewind_ret` output parameter to `-1` and does nothing else.
```
Sorry, copy-paste error in my previous suggestion.